src/OrchardCore.Modules/Orchard.Setup/Controllers/SetupController.cs

```diff
@@ -128,7 +129,8 @@ public class SetupController : Controller
                 AdminEmail = model.Email,
                 AdminPassword = model.Password,
                 Errors = new Dictionary<string, string>(),
-                Recipe = selectedRecipe
+                Recipe = selectedRecipe,
+                RecipeExtraParams = Request.Form.ToDictionary(keyValue => keyValue.Key, keyValue => keyValue.Value)
```

Comment:
I am not sure it's a good idea to just take everything coming from the Form. Looks like a potential security issue to me.